aries_cloudagent/messaging/jsonld/routes.py

```diff
@@ -94,8 +95,9 @@ class Meta:
         Schema.from_dict(
             {
                 "creator": fields.Str(required=False),
-                "verificationMethod": fields.Str(required=False),
-                "proofPurpose": fields.Str(required=False),
+                "verificationMethod": fields.Str(required=True),
+                "proofPurpose": fields.Str(required=True),
+                "type": fields.Str(required=False),
             }
         )
     )
```

Comment:
This schema seems to be not used anymore. Should we remove it? @dbluhm @burdettadam

Reply:
Also be aware that there are checks further down the need to be in sync with the schema see [create_verify_data](https://github.com/hyperledger/aries-cloudagent-python/blob/b0f0d1ed4b844c2e4aa0d3822655b7881e1168ad/aries_cloudagent/messaging/jsonld/create_verify_data.py#L52). Also there is/was a _creator_ that becomes the _verificationMethod_ again.

Reply:
You're correct. This is in consistent. Either we allow both `creator` and `verificationMethod` but none is required in the schema and we raise an error later if none is provided or we remove the `creator` property. I'd vote for removing the property.

Reply:
Hm... The `DocSchema` should be used in the `VerifyRequestSchema`. I'm not sure why it's not -- merge artifact perhaps?

Reply:
I think I'm done. You want to have a look? :)